manifests/Mega Freight.yaml

```diff
@@ -0,0 +1,15 @@
+name: Mega Freight
+authors: 1010todd
+homepage: https://github.com/1010todd/Mega-Freight
+license: GPL-3.0-or-later
+version: 4632c34904b855c3de5b6ac68486b825aba40be3
+shortDescription: Adds several mostly civilian ships for you and npc merchants( and pirates) to use for more variety in human space.
+
+description: This plugin adds several civilian and some military ships to add some more variety to human space. Mostly focused around larger ships but also have smaller freighters and starter-level ships as well.
+url: https://github.com/1010todd/Mega-Freight/archive/4632c34904b855c3de5b6ac68486b825aba40be3.zip
+iconUrl: https://github.com/1010todd/Mega-Freight/4632c34904b855c3de5b6ac68486b825aba40be3/main/icon.png
+autoupdate:
+  type: commit
+  updateUrl: https://github.com/1010todd/Mega-Freight
```

Comment:
```suggestion
```

You can safely omit this line, since you already have a git repository as `homepage`.